base/libgit2/callbacks.jl

```diff
@@ -45,7 +45,7 @@ function user_abort()
           (Cint, Cstring),
           Cint(Error.Callback), "Aborting, user cancelled credential request.")
 
-    return Cint(Error.EAUTH)
+    return Cint(Error.EUSER)
```

Comment:
👍

Reply:
Just going to mention that [`EUSER`](https://github.com/JuliaLang/julia/blob/master/base/libgit2/error.jl#L13) is a generic error code which is suppose to be for callback usage. It doesn't formally mean the user cancelled the callback.